server/src/db/migrations/20210128100612_updateTrips.cjs

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
Non-blocking, but in the future, I'd suggest adding more description to your migration name than `updateTrips` -- what if you end up having to update "trips" in multiple ways? I'd suggest `addUserIdToTrips` instead.